packages/blob/src/index.ts

```diff
@@ -233,6 +388,14 @@ function getToken(putOptions?: BlobCommandOptions): string {
   return process.env.BLOB_READ_WRITE_TOKEN;
 }
 
+function getApiUrl(): string {
+  return (
+    process.env.VERCEL_BLOB_API_URL ||
+    process.env.NEXT_PUBLIC_VERCEL_BLOB_API_URL ||
```

Comment:
Should this env variable be added when linking stores now? Or was it already the default?

I am thinking: we may have to update our documentation/dashboard to surface this new env variable required for client uploads?

Reply:
Oh wait, that's mostly for us to test things I believe, the only value needed for customers would be the default one ('https://blob.vercel-storage.com/'), 👍 

The only needed doc update would be the contributing ones we have in Notion probably/the edge-functions part

Reply:
runbook updated 👍